windows/src/engine/kmcomapi/com/keyboardlanguages/keymankeyboardlanguageinstalled.pas

```diff
@@ -34,10 +34,11 @@ interface
   msctf;
 
 type
-  TKeymanKeyboardLanguageInstalled = class(TKeymanKeyboardLanguage, IKeymanKeyboardLanguageInstalled, IIntKeymanKeyboardLanguage)   // I4376
+  TKeymanKeyboardLanguageInstalled = class(TKeymanKeyboardLanguage, IKeymanKeyboardLanguageInstalled, IKeymanKeyboardLanguageInstalled2, IIntKeymanKeyboardLanguage)   // I4376
```

Comment:
What's the difference between `IKeymanKeyboardLanguageInstalled` and  `IKeymanKeyboardLanguageInstalled2`?

Reply:
`IKeymanKeyboardLanguageInstalled2` is a newer version of the `IKeymanKeyboardLanguageInstalled` interface (and inherits from it). Per COM, you should not modify an interface once it is published except by following [certain complex rules](https://docs.microsoft.com/en-us/windows/win32/rpc/the-versioning-theory-for-rpc-and-com) which make life more difficult for the client and the library, so instead we introduce a new version of the interface with the additional methods we need, which is easy to understand.

I guess in theory we could rename it to `IKeymanKeyboardLanguageInstalled14` to indicate which version the interface appears in, but sequential numbering is common practice.